_posts/platform/deployment/buildpacks/2000-01-01-ssh-key.md

```diff
@@ -78,3 +78,7 @@ The output of the deployment might end with an error:
 
 In such case, you need to add your public key to the remote service you need to
 access (e.g. GitHub or GitLab).
+
+## SSH Key Management Strategy
+
+Because the buildpack cannot provide a password, the provided SSH key cannot be encrypted. Thus our advice to fetch private GitHub repository is to setup a GitHub Deploy Key, unencrypted and read-only only for the target repositories you want to fetch.
```

Comment:
Maybe add a word about the same thing on GitLab as we will soon provide integration with GitLab?

Reply:
Oh, you're definitely right

Reply:
How is it called on GitLab?

Reply:
https://docs.gitlab.com/ee/ssh/#deploy-keys Deploy Keys